docs/developers/guides/automatic_coin_conversion.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 8
+-->
+
+# Automated Coin Conversion
+
+Learn how to the Automated Coin Conversion feature works. {synopsis}
+
+In their ERC-20 representation, assets can be used to interact with dApps using the EVM. This standard allows developers to build applications that are interoperable with other products and services. In their Native Coin representation, they can be transferred between accounts on Evmos and other Cosmos chains using IBC. They cannot, however, be used to interact with dApps on Evmos, as Native Coins are not supported by the EVM since they don’t implement the ERC-20 standard.
+
+In order to reduce end-user complexity, Evmos should only allow single-token representation use between IBC Coin and ERC-20s. Consequently, the Evmos team developed the Automated Coin Conversion feature to achieve this goal. It consists on converting incoming IBC vouchers to ERC-20s and modifying outgoing IBC transfers to convert ERC-20s to IBC Coins. This automated conversion occurs if, and only if, the appropriate token mapping was registered through governance. If the token pair is not registered, the IBC coin will be left as is.
+
+Please read on for further understanding of this feature scope and functionality.
+
+## Outbound transactions
+
+As an Evmos user, you may want to move your ERC-20 tokens onto another Cosmos chains. You may want to do this to use your tokens on dApps on other Cosmos chains. The automated coin conversion feature makes this operation smooth. You can send ERC-20 tokens via an IBC transfer with a single step. You can perform this operation using the Evmos [IBC transfer page](https://app.evmos.org/transfer). Under the hood, the protocol will atomatically make the conversion from ERC-20 token to IBC coin and perform the transfer to the desired Cosmos chain.
```

Comment:
> As an Evmos user ...

Note that this is referring to an end-user, which is a different user than the one the guide is targeting (developers). I would recommend providing information to end users regarding the integration of this feature:

- What are the prerequisites for integration? Do I need Metamask/WalletConnect/Keplr?
- Do I need to add/change any new components to my application / UI to support ERC20 transfers via IBC or does it work with the same IBC transfer logic?
- How are the ERC20s represented? How do I transfer them via IBC?